src/unix/linux_like/mod.rs

```diff
@@ -1617,6 +1622,10 @@ cfg_if! {
         pub const STATX_ALL: c_uint = 0x0fff;
         pub const STATX_MNT_ID: c_uint = 0x1000;
         pub const STATX_DIOALIGN: c_uint = 0x2000;
+        pub const STATX_MNT_ID_UNIQUE: c_uint = 0x4000;
+        pub const STATX_SUBVOL: c_uint = 0x_0000_8000;
+        pub const STATX_WRITE_ATOMIC: c_uint = 0x_0001_0000;
+        pub const STATX_DIO_READ_ALIGN: c_uint = 0x_0002_0000;
```

Comment:
This constant is not yet present in bionic, likely because it was introduced to Linux in 6.14, and the latest Android kernel is 6.12.